src/reader/mod.rs

```diff
@@ -226,7 +225,7 @@ impl JournalReader {
 
 		fields.insert(
 			"__CURSOR".to_string(),
-			cursor.to_string_lossy().into_owned(),
+			cursor.to_string_lossy().into_owned().as_bytes().to_vec(),
```

Comment:
I don't think we need to go from `CStr` → `String` → `Vec`. It should be possible to go directly from `CStr` → `Vec`.